wpimath/src/main/java/edu/wpi/first/math/ComputerVisionUtil.java

```diff
@@ -5,146 +5,34 @@
 package edu.wpi.first.math;
 
 import edu.wpi.first.math.geometry.Pose3d;
-import edu.wpi.first.math.geometry.Rotation2d;
-import edu.wpi.first.math.geometry.Rotation3d;
 import edu.wpi.first.math.geometry.Transform3d;
-import edu.wpi.first.math.geometry.Translation3d;
 
 public final class ComputerVisionUtil {
   private ComputerVisionUtil() {
     throw new AssertionError("utility class");
   }
 
   /**
-   * Algorithm from https://docs.limelightvision.io/en/latest/cs_estimating_distance.html Estimates
-   * range to a target using the target's elevation. This method can produce more stable results
-   * than SolvePNP when well tuned, if the full 6d robot pose is not required. Note that this method
-   * requires the camera to have 0 roll (not be skewed clockwise or CCW relative to the floor), and
-   * for there to exist a height differential between goal and camera. The larger this differential,
-   * the more accurate the distance estimate will be.
+   * Returns the robot's pose in the field coordinate system given an object's field-relative pose,
+   * the transformation from the camera's pose to the object's pose (obtained via computer vision),
+   * and the transformation from the robot's pose to the camera's pose.
    *
-   * <p>Units can be converted using the {@link edu.wpi.first.math.util.Units} class.
+   * <p>The object could be a target or a fiducial marker.
    *
-   * @param cameraHeightMeters The physical height of the camera off the floor in meters.
-   * @param targetHeightMeters The physical height of the target off the floor in meters. This
-   *     should be the height of whatever is being targeted (i.e. if the targeting region is set to
-   *     top, this should be the height of the top of the target).
-   * @param cameraPitchRadians The pitch of the camera from the horizontal plane in radians.
-   *     Positive values up.
-   * @param targetPitchRadians The pitch of the target in the camera's lens in radians. Positive
-   *     values up.
-   * @param targetYawRadians The yaw of the target in the camera's lens in radians.
-   * @return The estimated distance to the target in meters.
+   * @param objectInField An object's field-relative pose.
+   * @param cameraToObject The transformation from the camera's pose to the object's pose. This
+   *     comes from computer vision.
+   * @param robotToCamera The transformation from the robot's pose to the camera's pose. This can
+   *     either be a constant for a rigidly mounted camera, or variable if the camera is mounted to
+   *     a turret. If the camera was mounted 3 inches behind the "origin" (usually physical center)
+   *     of the robot, this would be new Transform3d(Units.inchesToMeters(3.0), 0.0, 0.0, new
```

Comment:
Javadoc: Shouldn't this be `Units.inchesToMeters(-3.0)` now?

Reply:
Yep. The transform was inverted and we forgot to update the example in the parameter docs.